src/main/java/org/jvnet/hudson/test/RealJenkinsRule.java

```diff
@@ -646,16 +664,30 @@ public void doStatus(@QueryParameter String token) {
             System.err.println("Checking status");
             checkToken(token);
         }
+        /**
+         * Used to run test methods on a separate thread so that code that uses {@link Stapler.getCurrentRequest}
```

Comment:
either
```suggestion
         * Used to run test methods on a separate thread so that code that uses {@code Stapler.getCurrentRequest}
```
or
```suggestion
         * Used to run test methods on a separate thread so that code that uses {@link Stapler#getCurrentRequest}
```
after `import`ing it.

Reply:
Good catch, I fixed this in [869ed42](https://github.com/jenkinsci/jenkins-test-harness/pull/422/commits/869ed42113bfc9b6f3d3a4f70e26fa1841e7fc3e).